app/assets/javascripts/listings/templates/listing/_lottery_modal.html.slim

```diff
@@ -0,0 +1,26 @@
+span ng-controller="ListingController"
+  section.results-container.row.collapse
+    .medium-6.medium-push-6.columns
+      hgroup.padding--2x
+        span.t-small.t-bold Lottery results for
+        h2#modalTitle
+          | {{listing.Building_Name}}
+        ul.t-small.margin-bottom
+            li ng-repeat="(unit_type, units) in listing.Units | groupBy: 'Unit_Type'"
+               | {{unit_type || 'N/A'}} {{units.length}} unit(s)
+        p.t-small.t-bold A total of 900 tickets were drawn in this lottery.
```

Comment:
@chrisdolendo I'm assuming the "900 tickets" should actually be `{{listing.Lottery_Members.length}}` or something like that?